src/components/molecules/EarthEditor/PropertyPane/PropertyField/CameraField/index.tsx

```diff
@@ -262,10 +262,13 @@ const Popup = styled.ul<{ open: boolean }>`
 
 const FormGroup = styled.div`
   display: flex;
+  align-item: center;
 `;
 
 const FormIcon = styled(Icon)`
-  margin: 10px 10px 10px 0;
+  margin-top: auto;
+  margin-bottom: 25px;
+  width: 14.07%;
```

Comment:
If there is no support for resizing/mobile, etc please use pixels instead of percentage! 
But, in addition since you are modifying an Icon in this case, please use the Icon's size prop (as a pixel value)

Reply:
as dicused before the width property doesn't work right in this case unless I change the width of the popup itself , removing marging or padding affects only few pixels and I don't wan't to make any changes on icon componenent itself , so I suggest anther solution would please check , also I replace percentage with pixels